crawl-ref/source/mon-death.cc

```diff
@@ -2307,16 +2311,20 @@ item_def* monster_die(monster& mons, killer_type killer,
 
                 if (have_passive(passive_t::mp_on_kill))
                 {
+#if TAG_MAJOR_VERSION == 34
```

Comment:
the multiple `#ifdef` here is ugly, but I couldn't think of a way to do it with a single block, and it seemed brittle to duplicate the entire logic.

Reply:
You could write:

```
mp_heal = 1 + random2(mons.get_experience_level() / 2);

#if ...
  switch(...) {
    case GOD_PAKELLAS: mp_heal = whatever;
      break;
    default:
      break;
}
#endif
```

Reply:
thanks, good idea. I even removed the switch